parity/cli.rs

```diff
@@ -106,7 +107,7 @@ API and Console Options:
                            interface. APIS is a comma-delimited list of API
                            name. Possible name are web3, eth, net, personal,
                            ethcore, ethcore_set, traces.
-                           [default: web3,eth,net,ethcore,personal,traces].
+                           [default: web3,eth,net,ethcore,personal,traces,rpc].
```

Comment:
it's a meta-api, probably no reason to configure it in the cli, it should be just silently included
imo